doc/API.md

```diff
@@ -647,7 +633,7 @@ path string
 
```

Comment:
mentioned elsewhere, but I feel like 'path' really isn't a "parameter", those are supposed to mean query string parameters. 'path' is a url parameter, which I feel like should have a different section.

It is pretty unambiguous, but... hmm